MIT_LICENSE.txt

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2015-2023, MIT
```

Comment:
This file is not in the recipe directory. Do you need it?

Reply:
yep got rid of it. All changes look good now?

Reply:
are there other steps I need to take now that the pr is merged?